backend/DAL/UserDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
Dtos should not be located here. You need to create a separate folder for dtos with subfolders request and response. Outside of dal. But the entities should be here. And create a folder for repositories. And services/repositories should contain a subfolder for interfaces